packages/nl-design-system-unstable/status-badge-component/README.md

```diff
@@ -0,0 +1,79 @@
+# Status badge
+
+*Status: **In development***
+
+## Introduction
+
+A status badge draws attention and alerts a user to system states. The information in a status badge is short, concise and clearly scannable.
+
+## Variants
+
+- Success
+- Warning
+- Error
```

Comment:
I don't think "warning" is halfway on a scale between "success" and "error", but we can get into that at some later stage as a separate user story.

Reply:
Agreed, but let's indeed determine extra 'status' variations in a later stage.